### PR TITLE
Separation of PolarDistribution, Update docs

### PR DESCRIPTION
**Pull request by gwm17**
Split polar angle distribution to its own protocol, and update everything accordingly.

Update docs. Bump version to 0.1.5

### CODE REVIEW

**Comment by gwm17** [comment]
Resolves #20

File: docs/faq.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ In general attpc_engine is flexible, and can generate data for many different ty
 See [Converting to Dataframes](./user_guide/kinematics/index.md#converting-to-dataframes)
 
 ## How can make attpc_engine output data into a different format
```
Comment (sigmanotation) [inline docs/faq.md:18]: Should this be "How to make..."?

File: src/attpc_engine/kinematics/angle.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+from numpy.random import Generator
+import numpy as np
+
+
+class PolarDistribution(Protocol):
+    """Protocol for defining a reaction polar angle distribution in the CM frame
+
+    Note that for polar angle distributions, the coordinate to sample
+    is cos(polar) not polar (to cover solid angle uniformly). This means
+    hat the valid domain of these distributions is [0.0, pi]
```
Comment (sigmanotation) [inline src/attpc_engine/kinematics/angle.py:11]: I think this should be "that the valid..."

File: src/attpc_engine/kinematics/angle.py
Comment (sigmanotation) [inline src/attpc_engine/kinematics/angle.py:1]: Looks good!

**Review by sigmanotation — APPROVED** [review]
Overall looks good. Excited to test it out!

File: docs/faq.md (hunk 1 above)
Comment (gwm17) [reply docs/faq.md]: Fixed

File: src/attpc_engine/kinematics/angle.py (hunk 2 above)
Comment (gwm17) [reply src/attpc_engine/kinematics/angle.py]: Fixed

**Pull request merged**